modules/serverless-metering-reports.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+// serverless-metering.adoc
+
+[id="reports-metering-serverless_{context}"]
+== Running a metering report for Knative Serving
+You can run metering reports against Knative Serving by creating `Report` resources.
+Before you run a report, you must modify the input parameter within the `Report` resource to specify the start and end dates of the reporting period.
+
+Once you have provided the input parameters, you can run the report using the command:
+----
+$ oc apply -f reports/<report-name>.yml
+----
+You can then check the report as shown in the following example:
+----
+$ kubectl get report
+
+NAME                        QUERY                       SCHEDULE   RUNNING    FAILED   LAST REPORT TIME       AGE
+knative-service-cpu-usage   knative-service-cpu-usage              Finished            2019-06-30T23:59:59Z   10h
+----
+
+
+[id="knative-service-cpu-usage-report_{context}"]
+== knative-service-cpu-usage-report
+
+.YAML file
```

Comment:
Should these move further up possibly? I would've expected them below the paragraph explaining the parameters and before applying the YAML.